roles/ci_gen_kustomize_values/templates/include/ip-version.j2

```diff
@@ -0,0 +1,17 @@
+{%- if cifmw_ci_gen_kustomize_values_primary_ip_version == 4 -%}
```

Comment:
you might need to get a header comment such as:
`# source: include/ip-verison.j2`
or something like that in order to get a specific linter to pass:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/scripts/check_k8s_snippets_comment.sh

Reply:
ack, let's see if it is more happy now.

Reply:
Marking as resolved - something went sideways with the import / include approach.
I moved the map building to the vars instead.